core/rio/n3/src/main/java/org/eclipse/rdf4j/rio/n3/N3Parser.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
This is not the correct copyright header I'm afraid. It should be this:

```
/*******************************************************************************
 * Copyright (c) 2020 Eclipse RDF4J contributors.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *******************************************************************************/
```

See https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#code-formatting for details.